base-math/truncddsf.c

```diff
@@ -38,16 +41,29 @@ CONVERT_WRAPPER(
 	_Decimal64 a_norm;
 	long long mant;
 	int exp, sexp;
-	
+
```

Comment:
I'd prefer that this change, which is not related to the rounding direction mode problem, not be integrated.

Reply:
I agree. But notice that for base-math/trunctddf.c there's no tab on those lines. Maybe I should do another PR to fix that.